client/app/components/composites/ListingCard/ListingCard.story.js

```diff
@@ -78,6 +77,13 @@ const ListingCardNoImage =
           value: 0.67, // eslint-disable-line no-magic-numbers
           unit: ':km',
         }),
+        author: {
+          familyName: 'family name',
+          givenName: 'given name',
+          description: 'product author',
+          avatarURL: 'https://placehold.it/40x40',
+          profileURL: `#profile${Math.random(10)}`, // eslint-disable-line no-magic-numbers
+        },
```

Comment:
Yet another place that's clashing with the avatar profile pic changes, we need to have a central place for generating these :/